src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannelFactory.cs

```diff
@@ -47,6 +48,7 @@ public IRpcWorkerChannel Create(string scriptRootPath, string runtime, IMetricsL
                 throw new InvalidOperationException($"WorkerCofig for runtime: {runtime} not found");
             }
             string workerId = Guid.NewGuid().ToString();
+            _eventManager.AddGrpcChannels(workerId); // prepare the inbound/outbound dedicated channels
```

Comment:
@pragnagopa can you confirm this is being called during placeholder mode and shouldn't be impacting cold start?

Also how is this going to impact isolated cold start when there are no placeholders?

Reply:
@pragnagopa poke for update :)

Reply:
@safihamid -- Can you take this and run it in a private stamp to make sure there's no cold start regressions?

Reply:
ok I don't see any changes in cold start in my private stamp with this. 

this method (_eventManager.AddGrpcChannels(workerId))gets called during placeholder mode.